src/cov.jl

```diff
@@ -48,76 +49,60 @@ function scattermat end
 
 
 """
-    cov(X, wv::WeightVec; mean=nothing, vardim=1)
+    cov(X, wv::AbstractWeights; vardim=1, corrected=false)
 
-Compute the weighted covariance matrix. By default, the covariance
-matrix is normalized by the sum of the weights. That is, `cov(X, wv)`
-is equivalent to `scattermat(X, wv) / sum(wv)`.
+Compute the weighted covariance matrix. Similar to `var` and `std` the biased covariance
+matrix (`corrected=false`) can be computed by multiplying `scattermat(X, wv)` by
+``\\frac{1}{\\sum{w}}`` to normalize. However, the unbiased covariance matrix
```

Comment:
`w` has not been defined. Maybe replace `wv` with `w` everywhere it makes sense?